source/index.md

```diff
@@ -68,6 +68,28 @@ The following illustrate of what can be achieved with the Nix ecosystem:
 - Atomic upgrades and rollbacks.
 
 
+## Who is Nix for?
+
+Nix is a tool for software developers.
+
+You don't have to be a professional and you don't need formal education in informatics to greatly benefit from Nix.
```

Comment:
This sentence should come after the next one: first clarify what we mean by software developer, then make this statement that you don't have to be a professional and so on...

Reply:
Yeah, I was shuffling around those sentences until they stopped making sense to me. You're probably right.